libs/auth/sign-in/src/lib/components/google-button-login/types.ts

```diff
@@ -0,0 +1,33 @@
+import { ReactElement } from 'react';
```

Comment:
Types has been updated, see code on the master

https://github.com/instrybutor/pasnik/blob/master/apps/frontend/src/app/GoogleButtonLogin/types.ts

Reply:
fixed